build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+cd protos/spike
+curl -o spike.proto https://raw.githubusercontent.com/meateam/spike-service/master/proto/spike-service/spike.proto
+cd ../../
+npx grpc_tools_node_protoc --js_out=import_style=commonjs,binary:./protos/spike/generated --grpc_out=./protos/spike/generated -I ./protos ./protos/spike/spike.proto
```

Comment:
This is unrelated to `spike.proto`, but to `users.proto`, for future recompilation of the `users.proto` to Go and JS/TS, please add the commands on how to do so in the README.md (which for some reason is in `src` folder and not the root of the project, and is empty but still there...).
An example for the command to compile it for Go can be found in [Upload-Service README.md](https://github.com/meateam/upload-service), and can be found in other Go gRPC services we have.

Reply:
done.